src/integrations/faq.md

```diff
@@ -112,6 +112,16 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
 ### Where can I find network details such as deployed smart contract addresses? 
 In the [`network defaults`](https://github.com/nymtech/nym/blob/release/{{platform_release_version}}/common/network-defaults/src/mainnet.rs) file. 
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+The Circulating Supply is the number of available `NYM` tokens in the Nyx blockchain system that are often used by exchanges.
```

Comment:
* what are often used by exchanges? this sentence doesn't grammatically make sense. I don't think you need to mention that its often used by exchanges.. people who are looking for the circulating supply will know what it is when looking for it. 
* remove 'system' 
* its not just on the NYX chain is it - there's also the NYM on Ethereum as ERC20s.